src/ORM/Rule/ExistsIn.php

```diff
@@ -98,6 +109,11 @@ public function __invoke(EntityInterface $entity, array $options)
             return true;
         }
 
+        if ($this->_options['allowSqlNulls'] === true
+            && $this->_checkPartialSchemaNulls($entity, $source) === true
+        ) {
+            return true;
```

Comment:
> // Does this SiteCategory reference an existing Site.
> $rules->add($rules->existsIn(['site_id'], 'Sites'));
> // Does this SiteCategory reference and existing Parent SiteCategory or none at all (parent_id = null)
> $rules->add($rules->existsIn(['parent_id', 'site_id'], 'ParentNodes', ['allowSqlNulls' => true]));

@ionas Sorry, i cannot understand yet :cry: How does the above behave differently from the following rules?
```php
$rules->add($rules->existsIn(['site_id'], 'Sites'));
$rules->add($rules->existsIn(['parent_id'], 'ParentNodes'));
```